src/test/java/org/jabref/logic/formatter/bibtexfields/ShortenDOIFormatterTest.java

```diff
@@ -0,0 +1,24 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@FetcherTest
+class ShortenDOIFormatterTest {
+
+    private ShortenDOIFormatter sut;
```

Comment:
we try to avoid abbreviations as much as possible. `formatter` would be a better name.

Reply:
Sure. Fixed.